js/widgets/musickeyboard.js

```diff
@@ -1158,13 +1307,13 @@ function MusicKeyboard() {
 
         docById("wheelDivptm").style.left =
             Math.min(
-                this._logo.blocks.turtles._canvas.width - 200,
-                Math.max(0, x * this._logo.blocks.getStageScale())
+                logo.blocks.turtles._canvas.width - 200,
```

Comment:
Same here, can directly access - `turtles._canvas.width` but we have a getter for `_canvas` in `Turtles` class. We should use that instead of accessing private properties directly. Please look for other issues like that in the file and try fixing them.